### DISCUSSION TITLE
How to disable options ?

### DISCUSSION THREAD

**Discussion by pablom89** [umaranis/svelte-lexical#86] [Q&A] [answered]
Is there a way in SvelteKit to disable some options of the editor? Let's suppose I don't want the option to upload images or to italicize the text, for example?

**Comment by umaranis** [comment, marked as answer]
Hi @pablom89 ,

You can specify which options to include in the toolbar. See a sample toolbar code [here](https://github.com/umaranis/svelte-lexical/blob/master/demos/playground/src/ToolbarPlayground.svelte) . You can add/remove buttons from the toolbar. For instance, if you want to have image support, you can remove the option from the toolbar. But images will still be supported (for instance, one can paste an image into the editor). To remove image support altogether, you can remove the `ImagePlugin` and `ImageNode` from the editor (see sample [editor definition here](https://github.com/umaranis/svelte-lexical/blob/master/demos/playground/src/RichTextComposer.svelte))

**Reply by pablom89** [reply]
Thanks!
